go/store/prolly/map.go

```diff
@@ -196,6 +196,21 @@ func (m Map) iterFromRange(ctx context.Context, rng Range) (*orderedTreeIter[val
 	return treeIterFromRange(ctx, m.tuples.root, m.tuples.ns, rng)
 }
 
+// ConvertToKeylessIndex converts the given map to a keyless index map.
+func ConvertToKeylessIndex(m Map) Map {
```

Comment:
let's move this to `shim.go`, we probably want to revisit how we construct these tuple descriptors